### DISCUSSION TITLE
Changing Tech stack proficiency

### DISCUSSION THREAD

**Discussion by lucasfth** [s1lvax/route#36] [General]
I may have missed it. But is it possible right now to change the proficiency of the tech stack you have written, or is the only option to delete it and re-add it with the new proficiency?
<img width="583" alt="image" src="https://github.com/user-attachments/assets/c345d3e2-af2b-4ba4-a69d-e8b7a2e82dd8">
So on the above image would I be able to change C# to be `Proficient` without removing it and re-adding it?

**Comment by s1lvax** [comment]
> I may have missed it. But is it possible right now to change the proficiency of the tech stack you have written, or is the only option to delete it and re-add it with the new proficiency?

First of all, thank you for showing interest and welcome !

You're indeed correct, editing Skills and the Tech Stack has not been developed yet, but it's on the to-do list ! 

I thought I could delay it a little bit because there are just 2 fielded forms. The possibility to re-arrange the links/skills is also coming.

**Reply by lucasfth** [reply]
Sounds good. Hopefully, I will also find some time to make a contribution to the project at some point.

**Reply by s1lvax** [reply]
> Sounds good. Hopefully, I will also find some time to make a contribution to the project at some point.

You're welcome too anytime ! I hope we can build an awesome platform as a community!
